analysis_driver/reader/sample_sheet.py

```diff
@@ -48,6 +48,22 @@ def _read_sample_sheet(sample_sheet):
     f.close()
     return None, None
 
+def _remove_samplesheet_barcodes(data_dir):
+    with open((os.path.join(data_dir, 'SampleSheet_analysis_driver.csv'))) as openfile:
+        with_barcode = openfile.read()
+        data = ((with_barcode.split('[Data],\n')[1]).split('\n'))[1:]
+        barcodes = []
```

Comment:
Try to rewrite this using csv.DictReader, csv.DictWriter [See code in _populate](https://github.com/EdinburghGenomics/Analysis-Driver/blob/master/analysis_driver/reader/sample_sheet.py#L128)